app/Libraries/Markdown/OsuMarkdown.php

```diff
@@ -91,6 +91,8 @@ class OsuMarkdown
                 'html_input' => 'allow',
             ],
             'osu_extension' => [
+                'attributes_allowed' => ['flag', 'id'],
```

Comment:
does this also resolve https://github.com/ppy/osu-web/issues/9118 by any chance?